src/app/_components/RecentBlocks/BtcBlock.tsx

```diff
@@ -110,11 +111,36 @@ export function BtcBlock({ burnBlock }: { burnBlock: BurnBlock }) {
 
 export function NewestBtcBlock({ burnBlock }: { burnBlock: BurnBlock }) {
   const network = useGlobalContext().activeNetwork;
+
+  const { colorMode } = useColorMode();
```

Comment:
NewestBtcBlock style bugfix

Reply:
are we not able to use this notation? - {{ base: 'red', _dark: 'blue' }}
if not, why not just use semantic tokens, namespacing them as something like recentblocks: {btcblock: {...}}